packages/lesswrong/components/ultraFeed/UltraFeed.tsx

```diff
@@ -151,7 +153,24 @@ const UltraFeedContent = () => {
   
   const [settings, setSettings] = useState<UltraFeedSettingsType>(getStoredSettings);
   const [settingsVisible, setSettingsVisible] = useState(false);
-  const [sessionId] = useState(() => randomId());
+  const [sessionId] = useState<string>(() => {
+    if (!isClient) return randomId(); 
```

Comment:
I am pretty sure you don't need this check because `useState` doesn't get run during SSR.